package.json

```diff
@@ -66,7 +66,7 @@
     "react-dom": "15.4.1",
     "react-gemini-scrollbar": "2.1.5",
     "react-intl": "2.2.3",
-    "react-jsonschema-form": "0.50.0",
+    "react-jsonschema-form": "1.0.4",
```

Comment:
Any breaking changes to be concerned about in this upgrade?

Reply:
There was one, it doesn't look it affects us at all.